onnx/helper.py

```diff
@@ -320,3 +321,19 @@ def printable_graph(graph, prefix=''):
     # closing bracket
     content.append(prefix + '}')
     return '\n'.join(content)
+
+
+def strip_doc_string(proto):
```

Comment:
fancy

Reply:
figured it's faster to do this way than iterate over all fields manually :)